src/org/ohdsi/usagi/ui/DataChangeListener.java

```diff
@@ -17,19 +17,22 @@
 
 public interface DataChangeListener {
 
-	public static DataChangeEvent	APPROVE_EVENT		= new DataChangeEvent(true, false);
-	public static DataChangeEvent	SIMPLE_UPDATE_EVENT	= new DataChangeEvent(false, false);
-	public static DataChangeEvent	RESTRUCTURE_EVENT	= new DataChangeEvent(false, true);
+	DataChangeEvent	APPROVE_EVENT		= new DataChangeEvent(true, false, false);
+	DataChangeEvent	SIMPLE_UPDATE_EVENT	= new DataChangeEvent(false, false, false);
+	DataChangeEvent	MULTI_UPDATE_EVENT	= new DataChangeEvent(false, false, true);
+	DataChangeEvent	RESTRUCTURE_EVENT	= new DataChangeEvent(false, true, false);
```

Comment:
If you don't need other calls to `DataChangeEvent` than the ones listed above, please consider changing `DataChangeEvent` to an enum, with the constants above and the same fixed properties. 

Having constants in an interface is fine, but these are not constants. I'm not sure if interface requires the `final` qualifier to ensure it is not reassigned. More to the point, the `DataChangeEvent` class should be immutable to be a constant (`final` in each property). If `DataChangeEvent` properties are intentionally mutable, it would be better to replace these references with factory functions:
```java
default DataChangeEvent createApproveEvent() {
  return new DataChangeEvent(true, false, false);
}
```